core/node/libp2p/rcmgr_defaults.go

```diff
@@ -132,9 +140,29 @@ func createDefaultLimitConfig(cfg config.SwarmConfig) (rcmgr.LimitConfig, error)
 			StreamsOutbound: 0,
 		},
 
-		// Just go with what libp2p does
-		TransientBaseLimit:     rcmgr.DefaultLimits.TransientBaseLimit,
-		TransientLimitIncrease: rcmgr.DefaultLimits.TransientLimitIncrease,
+		TransientBaseLimit: rcmgr.BaseLimit{
+			Streams:         bigEnough,
+			StreamsInbound:  rcmgr.DefaultLimits.TransientBaseLimit.StreamsInbound,
+			StreamsOutbound: bigEnough,
+			Conns:           bigEnough,
+			ConnsInbound:    rcmgr.DefaultLimits.TransientBaseLimit.ConnsInbound,
+			ConnsOutbound:   bigEnough,
+			FD:              rcmgr.DefaultLimits.TransientBaseLimit.FD,
+			Memory:          rcmgr.DefaultLimits.TransientBaseLimit.Memory,
```

Comment:
```suggestion
			Memory:          rcmgr.DefaultLimits.TransientBaseLimit.Memory,
			FD:              rcmgr.DefaultLimits.TransientBaseLimit.FD,
			
			Conns:           bigEnough,
			ConnsInbound:    rcmgr.DefaultLimits.TransientBaseLimit.ConnsInbound,
			ConnsOutbound:   bigEnough,
			
			Streams:         bigEnough,
			StreamsInbound:  rcmgr.DefaultLimits.TransientBaseLimit.StreamsInbound,
			StreamsOutbound: bigEnough,
```

nit: putting in the same ordering as used throughout the code.